HOW_TO_RELEASE.md

```diff
@@ -0,0 +1,20 @@
+# How To Release the `metalearn` Package
+
+### Testing
+
+1. Using a clean environment, do a final test of the current state of the `develop` branch. Make sure before tests are run to delete or move the `metalearn` directory so the tests are run against the installed version of `metalearn` and not the local files.
+1. Test using the develop branch of the `metalearn` package with our `d3m_primitives` package, to make sure the APIs are compatible.
```

Comment:
Should we link the `d3m_primitives` package?

Reply:
That's a good idea, I'll do that.

Reply:
Finished.